pkgs/applications/radio/chirp/default.nix

```diff
@@ -38,6 +38,7 @@ python3.pkgs.buildPythonApplication rec {
 
   passthru.updateScript = unstableGitUpdater {
     branch = "py3";
+    hardcodeZeroVersion = true; # tags are "release_x_y_z" which requires more logic in unstableGitUpdater to convert, last release was 0.4.0 in 2014
```

Comment:
This *could* be an argument for some logic in `unstableGitUpdater` itself to convert the tag based on a requested *thing* (pattern, code, whatever) into a regular `major[.minor[.patch]][whatever]`, but that is likely overkill (or at least, way more than what my energy levels permit me to do).

Maybe something like what `goperf` does where it first runs `unstableGitUpdater`, followed by some more code to update further things? The regular updater could write `x_y_z-unstable-YYY-MM-DD`, and the extra code could fix it up into `x.y.z-unstable-YYYY-MM-DD`?

Also, the `py3` branch seems to have been deleted so this still doesn't work, but I'm not sure what it should point at.